uwrt_mars_rover_hw/drivers/roboteq_driver/src/CanopenInterface.cpp

```diff
@@ -254,36 +263,46 @@ DataType CanopenInterface::sendQuery(RuntimeQuery query, uint8_t subindex) {
 
   ssize_t bytes_written = write(roboteq::CanopenInterface::socket_handle_, &query_frame, sizeof(struct can_frame));
   if (bytes_written != sizeof(struct can_frame)) {
-    // TODO: throw error
+    throw StringException("Need more bytes to write");
     return 0;
   }
 
   struct can_frame response_frame = {};
   ssize_t bytes_read = read(roboteq::CanopenInterface::socket_handle_, &response_frame, sizeof(struct can_frame));
 
-  //  std::cout << std::hex << response_frame.can_id << "\t" << static_cast<unsigned>(response_frame.can_dlc) << "\t"
-  //            << static_cast<unsigned>(response_frame.data[0]) << "\t" <<
-  //            static_cast<unsigned>(response_frame.data[1])
-  //            << "\t" << static_cast<unsigned>(response_frame.data[2]) << "\t"
-  //            << static_cast<unsigned>(response_frame.data[3]) << "\t" <<
-  //            static_cast<unsigned>(response_frame.data[4])
-  //            << "\t" << static_cast<unsigned>(response_frame.data[5]) << "\t"
-  //            << static_cast<unsigned>(response_frame.data[6]) << "\t" <<
-  //            static_cast<unsigned>(response_frame.data[7])
-  //            << std::endl;
+   std::cout << std::hex << response_frame.can_id << "\t" 
```

Comment:
Should turn this into a ROS_DEBUG_STREAMED

Reply:
bump

Reply:
Done
```
  ROS_DEBUG_STREAMED(<< std::hex << response_frame.can_id << "\t" << static_cast<unsigned>(response_frame.can_dlc)
                     << "\t" << static_cast<unsigned>(response_frame.data[0]) << "\t"
                     << static_cast<unsigned>(response_frame.data[1]) << "\t"
                     << static_cast<unsigned>(response_frame.data[2]) << "\t"
                     << static_cast<unsigned>(response_frame.data[3]) << "\t"
                     << static_cast<unsigned>(response_frame.data[4]) << "\t"
                     << static_cast<unsigned>(response_frame.data[5]) << "\t"
                     << static_cast<unsigned>(response_frame.data[6]) << "\t"
                     << static_cast<unsigned>(response_frame.data[7]));
```